pyfar/dsp/dsp.py

```diff
@@ -143,6 +143,50 @@ def wrap_to_2pi(x):
     return x
 
 
+def linear_phase(signal, group_delay):
```

Comment:
What about a unit parameter, so `group_delay` could be defined in seconds or milliseconds?

Reply:
Good idea, done as suggested